src/Package/Impl/Plots/PlotContentProvider.cs

```diff
@@ -31,7 +31,9 @@ internal sealed class PlotContentProvider : IPlotContentProvider {
         private async void RSession_Connected(object sender, EventArgs e) {
             // Let the host know the size of plot window
             if (_lastPixelWidth >= 0 && _lastPixelHeight >= 0) {
-                await ApplyNewSize();
+                try {
+                    await ApplyNewSize();
+                } catch (OperationCanceledException) { } catch (MessageTransportException) { } catch (RException) { }
```

Comment:
It's for R library errors like "invalid graphics state" and such.

From DoNotWait:
```
        public static void DoNotWait(System.Threading.Tasks.Task task) {
            // Errors like invalid graphics state which go to the REPL stderr will come back
            // in an Microsoft.R.Host.Client.RException, and we don't need to do anything with them,
            // as the user can see them in the REPL.
            task.SilenceException<MessageTransportException>()
                .SilenceException<Microsoft.R.Host.Client.RException>()
                .DoNotWait();
        }
```